app/components/floating-button/FloatingButton.js

```diff
@@ -37,6 +37,10 @@ class FloatingButton extends Component {
 
   handleClose = () => this.setState({ isShowingActions: false })
 
+  showComposer(type) {
+    return this.props.showComposer.bind(this, type)
```

Comment:
Oh! You're right! It's very very late and for some reason I was thinking about reassigning the method on `this.props` instead of just on `this`. I'll change it. I'll stick to just using `bind` though as more people will immediately know what's going on. Thanks for the heads up :-)